go.mod

```diff
@@ -3,38 +3,30 @@ module github.com/ipfs/go-graphsync
 go 1.12
 
 require (
-	github.com/btcsuite/btcd v0.0.0-20190629003639-c26ffa870fd8 // indirect
-	github.com/gogo/protobuf v1.2.1
-	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/filecoin-project/go-data-transfer v0.0.0-20191219005021-4accf56bd2ce
```

Comment:
Not used? Probably needs a `go mod tidy` (or you may not have checked something in?).